apps/web/src/styles/globals.css

```diff
@@ -7,9 +7,55 @@
 @layer base {
   html,
   body {
-    @apply h-full;
+    @apply h-full text-base-textPrimary bg-base-bg;
     font-family: 'Inter', ui-sans-serif, system-ui, -apple-system !important;
   }
+
+  :root {
+    --textPrimary: 0, 0, 0;
+    --textSecondary: 34, 34, 34;
+    --textTertiary: 68, 68, 68;
+    --textDisabled: 102, 102, 102;
+
+    --bg: 236, 236, 236;
+    --bgCard: 255, 255, 255;
+    --bgPanel: 255, 255, 255;
+
+    --actionPrimary: 10, 17, 29;
+    --actionContrastText: 255, 255, 255;
+    --actionSecondary: 18, 220, 197; /* was "teal" */
+    --actionAccent: 2, 251, 194; /* was "green" */
```

Comment:
What is meant by this was teal? It still is, right?

Reply:
It may be helpful to document the colors next to each

Reply:
Ohhh right, that makes sense 💯

Reply:
re: documenting the colors... yeah, my biggest gripe with this implementation is that these variables aren't even a proper css color since they need to be interpolated. This makes telling what they are using your editor (or viewing/modifying them in chrome devtools) is pretty meh.

Here's the tailwind docs on it FWIW:
https://tailwindcss.com/docs/customizing-colors#using-css-variables

Reply:
Totally, that was my thought as well. I did see that was Tailwind's recommendation though, and it is a clean approach, just would be nice if you could list the actual colors.

Reply:
👍🏼 working on getting main merged in now and then I'll update the ones that aren't just some shade of gray